generators/app/templates/src/app/screens/Registration/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback } from 'react';
+import i18next from 'i18next';
+
+import FormInput from '~components/FormInput';
+import PATHS from '~components/Routes/paths';
+import { stringArrayToObject } from '~utils/array';
+
+import styles from './styles.module.scss';
+
+function RegistrationContainer() {
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  const handleSubmit = useCallback((event: React.FormEvent<Element>) => {
+    // TODO: Implement
+  }, []);
+
+  const handleInputChange = useCallback((event: React.FormEvent<Element>) => {
+    // TODO: Implement method
+  }, []);
+
+  const FIELDS = stringArrayToObject(['name', 'surname', 'email', 'password', 'confirmPassword']);
```

Comment:
this is a constant, move it outside the component so as to not redefine it in each render